File: APVN.LeadsPlatform.Models/LeadsQuantityForDealerIndexModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace APVN.LeadsPlatform.Models
{
    public class LeadsQuantityForDealerIndexModel : Pager
    {
        public string DealerEmail { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public List<LeadsQuantityForDealerModel> ListLeadsQuantityForDealer { get; set; }
    }
}

File: APVN.LeadsPlatform.Client/apvn-leadsplatform-client/src/app/models/LeadsQuantityForDealerIndexModel.ts
import { LeadsQuantityForDealerModel } from './LeadsQuantityForDealerModel';
import { Pager } from './ResponseData';

export class LeadsQuantityForDealerIndexModel extends Pager{
    constructor() {
        super();
        this.dealerEmail = '';
        this.listLeadsQuantityForDealer = [];
    }
    dealerEmail: string;
    startDate: Date;
    endDate: Date;
    listLeadsQuantityForDealer: LeadsQuantityForDealerModel[];
}